packages/mdc-menu-surface/_mixins.scss

```diff
@@ -30,6 +31,6 @@
   @include mdc-theme-prop(background-color, $color);
 }
 
-@mixin mdc-menu-surface-corner-radius($radius) {
-  border-radius: $radius;
+@mixin mdc-menu-surface-shape-radius($radius) {
```

Comment:
Is there a reason this doesn't expose/forward `$rtl-reflexive`?

Reply:
Added `rtl-reflexive` for menu surface.